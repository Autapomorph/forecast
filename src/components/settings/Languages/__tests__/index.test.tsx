import React from 'react';
import { shallow } from 'enzyme';

import { Languages } from '..';

describe('Languages tests', () => {
  const languages = [
    {
      title: 'title',
      code: 'code',
    },
    {
      title: 'title2',
      code: 'code2',
    },
  ];
  const props = {
    availableLanguages: languages,
    selectedLanguage: languages[0],
    _changeLanguage: jest.fn(),
  };

  it('renders properly', () => {
    const wrapper = shallow(<Languages {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
