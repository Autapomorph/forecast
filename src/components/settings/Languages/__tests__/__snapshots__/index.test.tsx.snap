// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Languages tests renders properly 1`] = `
<styles__SettingContainer>
  <styles__SettingTitle>
    Language
  </styles__SettingTitle>
  <styles__SettingContent>
    <Select
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "English",
            "value": Object {
              "code": "en-US",
              "title": "English",
            },
          },
          Object {
            "label": "Русский",
            "value": Object {
              "code": "ru-RU",
              "title": "Русский",
            },
          },
        ]
      }
      value={
        Object {
          "label": "title",
          "value": Object {
            "code": "code",
            "title": "title",
          },
        }
      }
    />
  </styles__SettingContent>
</styles__SettingContainer>
`;
